.github/workflows/reusable/agents-ai-changelog.yml

```
@@ -0,0 +1,71 @@
+        type: string
+        required: true
+      previous-tag:
+        description: Previous tag to diff against (empty = first release).
+        type: string
+        default: ''
+    secrets:
+      anthropic-api-key:
+        description: Anthropic API key.
+        required: true
+
+outputs:
```

actionlint (1.7.12)

[error] unexpected key "outputs" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)